docker/templates/README.md

```diff
@@ -0,0 +1,79 @@
+# 🍪 Containerization templates
+
+This directory contains [cookiecutters](https://github.com/audreyr/cookiecutter)
+for the containerization of web applications.
+
+## Getting started
+
+### 1. Set up.
+
+Install `cookiecutter` in your virtual environment of choice. (It's a [great
+candidate for `gus`](https://github.com/datamade/ops/wiki/How-to-DataMade-(Resources-for-new-hires)#python)!)
+
+```bash
+pip install cookiecutter
+```
+
+Then, clone this repository to your favorite working directory –
+
+```bash
+git clone https://github.com/datamade/how-to.git
+```
+
+– or update your local version.
+
+```bash
+cd /path/to/how-to
+git pull
+```
+
+### 2. Run `cookiecutter`.
+
+`cd` into your project directory. From here, we'll use `cookiecutter` templates
+to generate deployment scripts for your app or server.
+
+From your project directory, run `cookiecutter`:
+
+```bash
+cookiecutter how-to/docker/templates/python
+```
+
+#### Template variables
+
+When you run `cookiecutter`, it will ask you to define the following variables
+in your terminal:
+
+| Variable | Definition |
+| - | - |
+| `directory_name` | The directory that will contain your generated files. We'll move the files out and remove this directory after running `cookiecutter` so it's fine to use the default here. |
+| `app_name` | The name of your application. |
```

Comment:
Apps have a lot of different "names"  -- in particular the GitHub repo, Django app name and Django app label are all usually formatted differently. Maybe we could provide a clearer description, like `The slug you use to refer to your application (typically the same as the GitHub repo)`?